sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataView.java

```diff
@@ -114,5 +114,9 @@ public void enter(ViewChangeEvent event) {
 			
 		}
 		
+		Boolean isCaseEditAllowed  = FacadeProvider.getCaseFacade().isCaseEditAllowed(getCaseRef().getUuid());
```

Comment:
Try to move this logic into the AbstractCaseView. The CaseContactsView will need to override this. Maybe add a isCaseEditAllowed method to the AbstractCaseView that can be overriden by the CaseContactsView

Reply:
moved both to AbstractCaseView and AbstractContactView